src/main/java/wolox/training/models/User.java

```diff
@@ -22,8 +23,14 @@
     public User() {
     }
 
+    public User(String username, String name, LocalDate birthdate) {
```

Comment:
Same, you don't need a new constructor. Collection of books will never come in the json for creation.